binaries/org.eclipse.swt.win32.win32.x86_64/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Fragment-Host: org.eclipse.swt;bundle-version="[3.126.100,4.0.0)"
 Bundle-Name: %fragmentName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.swt.win32.win32.x86_64; singleton:=true
-Bundle-Version: 3.126.100.qualifier
+Bundle-Version: 3.127.000.qualifier
```

Comment:
```suggestion
Bundle-Version: 3.127.0.qualifier
```

I though the minor version was already incremented in this release?

Reply:
Yes, I just noticed the conflicts in this PR and I was about to fix them :-)

One sec

Reply:
Done